plugins/modules/ovirt_vm.py

```diff
@@ -906,14 +906,23 @@
         type: str
     virtio_scsi_enabled:
         description:
-            - "Enable Virtio SCSI support."
+            - "If I(true), the Virtio-SCSI devices will obtain a number of multiple queues depending on
+               the available virtual Cpus and disks, or according to the specified virtio_scsi_multi_queues."
         type: bool
         version_added: 1.7.0
     multi_queues_enabled:
         description:
             - "If `true`, each virtual interface will get the optimal number of queues, depending on the available virtual Cpus."
         type: bool
         version_added: 1.7.0
+    virtio_scsi_multi_queues:
+        description:
+            - "Number of queues for a Virtio-SCSI contoller, possible values:
+              -1 - Indicates that the queues will be automatically set, see virtio_scsi_enabled
+               0 - Indicates that the Virtio SCSI will be disabled (virtio_scsi_enabled false)
```

Comment:
it doesn't indicate that virtio-scsi will be disabled but rather that virtio-scsi multi-queue will be disabled (when virtio_scsi_enabled is set, we add the virtio-scsi controller and so we can add virtio-scsi disks)

Reply:
Fixed